.env

```diff
@@ -0,0 +1 @@
+PORT=3001
```

Comment:
Usually the `.env` file is not committed but a `.env.example` that can be copied as `.env`.

Reply:
Is that actually the situation in these React apps as well, at least in [create-react-app docs](https://create-react-app.dev/docs/adding-custom-environment-variables/#adding-development-environment-variables-in-env) they advice the other way.